.obs/dockerfile/slem-base-iso/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY --from=os / rootfs
 
 # Version value is taken form the elemental repository tags
 # Release value of this image and os image are unrelated
-RUN elemental --debug --config-dir . build-iso -o /output -n "sle-micro.$(uname -m)-base-%VERSION%-Build%RELEASE%" dir:rootfs
+RUN elemental --debug --config-dir . build-iso -o /output -n "sle-micro-base.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```

Comment:
The former iso name was not following the OBS pattern for naming images. Hence it was breaking the static links within the downloadable ISOs repository.

Without this PR in Staging (the same in Dev) we have:
```
# These are the _regular_ build (teal equivalent) including full version information
sle-micro.x86_64-2.0.0-Build15.4.iso
sle-micro.x86_64-2.0.0-Build15.4.iso.sha256
sle-micro.x86_64-2.0.0-Build15.4.iso.sha256.asc

# These are the static links of the base iso !! @_@ There is no base anywhere and version appears
sle-micro.x86_64-2.0.0.iso
sle-micro.x86_64-2.0.0.iso.sha256
sle-micro.x86_64-2.0.0.iso.sha256.asc

# These are the base builds including full version information
sle-micro.x86_64-base-2.0.0-Build5.6.iso
sle-micro.x86_64-base-2.0.0-Build5.6.iso.sha256
sle-micro.x86_64-base-2.0.0-Build5.6.iso.sha256.asc

# These are the static links to the _regular_ build without any version within the name
sle-micro.x86_64.iso
sle-micro.x86_64.iso.sha256
sle-micro.x86_64.iso.sha256.asc
```

Then adding this PR we have:
```
# These are the base builds including full version information
sle-micro-base.x86_64-2.0.0-Build7.2.iso
sle-micro-base.x86_64-2.0.0-Build7.2.iso.sha256
sle-micro-base.x86_64-2.0.0-Build7.2.iso.sha256.asc

# These are the static links of the base iso
sle-micro-base.x86_64.iso
sle-micro-base.x86_64.iso.sha256
sle-micro-base.x86_64.iso.sha256.asc

# These are the _regular_ build (teal equivalent) including full version information
sle-micro.x86_64-2.0.0-Build16.1.iso
sle-micro.x86_64-2.0.0-Build16.1.iso.sha256
sle-micro.x86_64-2.0.0-Build16.1.iso.sha256.asc

 These are the static links to the _regular_ build without any version within the name
sle-micro.x86_64.iso
sle-micro.x86_64.iso.sha256
sle-micro.x86_64.iso.sha256.asc
```